packages/@ama-sdk/core/src/fwk/clients/api-beacon-client.ts

```diff
@@ -37,25 +35,23 @@ const isPromise = <T>(value: T | Promise<T>): value is Promise<T> => value && ty
 export class ApiBeaconClient implements ApiClient {
 
   /** @inheritdoc */
-  public options = DEFAULT_OPTIONS;
+  public options;
```

Comment:
the type should be BaseApiBeacontClientOPtions ? Maybe it is not needed if we implement an interface :)